runtimes/pythonrt/worker_grpc_handler.go

```diff
@@ -91,6 +91,33 @@ func (s *workerGRPCHandler) IsActiveRunner(ctx context.Context, req *pb.IsActive
 	return &pb.IsActiveRunnerResponse{}, nil
 }
 
+func pushMsg(initialCall bool, ch chan *logMessage, m *logMessage) error {
+	if initialCall {
```

Comment:
initial call is implementation details of the pythonrt run 
please keep it there and not in this handler side